src/ledgerwriter/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
haven't seen this before, is this to navigate the final classes?

Reply:
It's to configure Mockito for final classes and methods. Reference here: https://www.baeldung.com/mockito-final